apis/v1alpha1/gatewayclass_types.go

```diff
@@ -126,12 +107,6 @@ type GatewayClassStatus struct {
 	// +kubebuilder:validation:MaxItems=8
 	// +kubebuilder:default={{type: "InvalidParameters", status: "Unknown", message: "Waiting for controller", reason: "Waiting", lastTransitionTime: "1970-01-01T00:00:00Z"}}
 	Conditions []metav1.Condition `json:"conditions,omitempty"`
```

Comment:
Can't really comment on the right diff but I think we now need to remove the Condition `GatewayReasonNamespaceForbidden` in gateway_types.go.

Reply:
Perhaps we should generalize that to `GatewayReasonForbidden`?

Reply:
I'm fine with that too but I would prefer to leave the condition out for now since the workflow of forbidden gateway is not clear yet.

Reply:
IMHO, either option (renaming or removing) is OK for now, but we should do one or the other :)

Reply:
Updated to just remove the condition out, we can always add one back in later.